pageserver/src/tenant/timeline.rs

```diff
@@ -1169,7 +1194,7 @@ impl Timeline {
                                " - keyspace={:?} lsn={}"),
                        seq_err, keyspace, lsn) },
             (Ok(_), Err(GetVectoredError::GetReadyAncestorError(GetReadyAncestorError::AncestorLsnTimeout(_)))) => {
-                // Sequential get runs after vectored get, so it is possible for the later 
```

Comment:
This was just a whitespace error, I tried to keep from adding it but it must've slipped at one point :)